Assets/JsonDotNet.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 9f042f1852f0c5548b3b681cd6086653
+guid: fedd074218b31794b950eee60e347590
```

Comment:
please revert

Reply:
oh nvm 

Seems like we need to update this branch with the latest dev stuff before we can review.